### PR TITLE
[WIP] Add External Dependency section

### PR DESCRIPTION
**Pull request by cnelson**
Add specific instructions in case of failures of our external dependencies as required by https://github.com/18F/cg-product/issues/311

First pass at this, still tweaking, but 👀 are welcome :)

cc: @mogul @dlapiduz @sharms @jmcarp

### CODE REVIEW

File: content/ops/contingency-plan.md [hunk 1]
```diff
@@ -88,6 +88,47 @@ If something goes wrong, we have these continuously maintained resources to supp
...
+* Migrate GitHub repositories from backups to GitLab
+* Update all Concourse jobs to retreieve resources from the newly provisioned Gitlab instance.
+
+After these steps are complete, updates will be deployed per policy using GitLab in place of GitHub.
+
+### Pager Duty
+Cloud Operations will configure all alerts to be delivered via email to [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov) if there is a disruption in Pager Duty service.
+
+### New Relic
+There is no direct mpact to the platform if a disruption occurs.  When debugging any issues where New Relic would provide insite, the team will use manual investigation to access the same information directly from the affected system(s).
```
Comment (mogul) [inline content/ops/contingency-plan.md]: s/insite/insight/

File: content/ops/contingency-plan.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Cloud Operations will postpone any non-critical updates to the platform until th
   * Perform the change on the local copy of the repository
   * Manually deploy the change by provisioning a Concourse jumpbox container, copying in the repository,  and executing any required steps by hand.
 
-When the distruption is resolved, Cloud Operations with push any changes made to the appropriate repositories in GitHub to restore it to the current known good state.
+When the distruption is resolved, Cloud Operations with push any changes made to the appropriate repositories in GitHub to restore them to the current known good state.  Cloud Operations will monitor Concourse to ensure it redploys any changes pushed to GitHub and then verify the system is in the expected state after all automated deployments complete.
```
Comment (mogul) [inline content/ops/contingency-plan.md]: s/distruption/disruption/

Comment (mogul) [reply content/ops/contingency-plan.md]: s/redploys/redeploys/

File: content/ops/contingency-plan.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ After these steps are complete, updates will be deployed per policy using GitLab
 Cloud Operations will configure all alerts to be delivered via email to [cloud-gov-support@gsa.gov](mailto:cloud-gov-support@gsa.gov) if there is a disruption in Pager Duty service.
 
 ### New Relic
-There is no direct mpact to the platform if a disruption occurs.  When debugging any issues where New Relic would provide insite, the team will use manual investigation to access the same information directly from the affected system(s).
+There is no direct mpact to the platform if a disruption occurs.  When debugging any issues where New Relic would provide insight, the team will use manual investigation to access the same information directly from the affected system(s).
```
Comment (mogul) [inline content/ops/contingency-plan.md]: s/mpact/impact/

**Comment by brittag** [comment]
Moving my thoughts from Slack: I added a few tiny edits to this new section, and this section looks good to me!

I suspect that an external person might look at the overall page and feel confused, since it talks about GitHub in a few places in slightly different ways (which I take responsibility for, since I wrote the earlier draft). We should eventually cut redundant aspects of this page, but I don't know how we want to prioritize working on that.

**Comment by mogul** [comment]
I'm gonna say it's good enough for now, could-be-better, but we have bigger fish to fry.

**Pull request merged**